posthog/models/filters/filter.py

```diff
@@ -89,6 +90,7 @@ class Filter(
     SearchMixin,
     DistinctIdMixin,
     EmailMixin,
+    UpdatedAfterMixin,
```

Comment:
offtopic: Having this many Mixins doesn't sit right with me, although I haven't dived into it to tell if it's just a smell or an actual issue. Wondering how a composition-based approach would look like :thinking: 

Regardless, nothing related to this PR.